src/shogun/mathematics/linalg/ratapprox/logdet/LogDetEstimator.cpp

```diff
@@ -180,28 +166,24 @@ SGMatrix<float64_t> CLogDetEstimator::sample_without_averaging(
 	index_t num_trace_samples = m_trace_sampler->get_num_samples();
 	SGMatrix<float64_t> samples(num_trace_samples, num_estimates);
 
-	#pragma omp parallel for
-		for (index_t i = 0; i < num_estimates; i++)
+	// TO DO: use openmp #pragma omp parallel for reduction(+:
+	// samples[:num_estimates][:num_trace_samples])
+	for (index_t i = 0; i < num_estimates; i++)
```

Comment:
same story... prefix vs post.
btw this code seems to be a bit of a copy-paste from the one above